src/resources/views/pages/articles/index.blade.php

```diff
@@ -3,6 +3,9 @@
 @push('pre-js')
     {{--    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>--}}
     {{--    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.js"></script>--}}
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
```

Comment:
moment.js is pretty big. Instead of downloading it from cdn we should compile it to app.js.
- Upgrade all packages: `yarn upgrade`
- Install with `yarn add moment`
- Require the module in `resources/js/app.js`: `let moment = require('moment');`
- Configure webpack to only keep `en-us` and `hu` locales: See example: https://momentjs.com/docs/#/use-it/webpack/